packages/pyolite-kernel/src/kernel.ts

```diff
@@ -44,7 +44,7 @@ export class PyoliteKernel extends BaseKernel implements IKernel {
 
     const { origin } = window.location;
 
-    const pypi = URLExt.join(origin, PageConfig.getOption('appUrl'), 'build/pypi');
+    const pypi = URLExt.join(origin, PageConfig.getOption('baseUrl'), 'build/pypi');
```

Comment:
could be [`getBaseUrl`](https://github.com/jupyterlab/jupyterlab/blob/f786d539ab5210464d7bc02cda88999949335b6e/packages/coreutils/src/pageconfig.ts#L106)...

Reply:
ah right good catch!